src/dcegm/solve.py

```diff
@@ -109,8 +110,18 @@ def solve_dcegm(
         compute_utility=compute_utility,
         compute_marginal_utility=compute_marginal_utility,
     )
+    (
+        state_choices_per_period,
+        states_per_period,
+    ) = determine_states_and_state_choices_per_period(
```

Comment:
Can you rename that to ```determine_states_and_state_choice_combs_per_period```?
Then it's consistent with our variable naming.

Reply:
Other function name, but lets abandon combs!